next/components/node/index.tsx

```diff
@@ -16,6 +17,9 @@ export function Node({ node }: { node: TypedRouteEntity }) {
     case "NodeArticle": {
       return <NodeArticle article={node} />;
     }
+    case "NodeTestContent": {
+      return <NodeTestContent testContent={node} />;
+    }
```

Comment:
We add a new case to the switch here to render our component in case we are rendering a node of type NodeTestContent